### ISSUE TITLE
rclpy is not importing module paths correctly

### ISSUE THREAD

**Issue by austinstig** [austinstig/ros2-raspberry-pi-zero-w#1]
https://answers.ros.org/question/299917/how-do-i-install-rclpy-on-an-arm-processor-with-ros2/

**Comment by pokitoz** [comment]
Did you get any update on that issue ?

**Comment by austinstig** [comment]
Yes and no. I got a cross compiler toolchain working but it looks like at the end of the day it was limited because I could not compile all of the dependencies to run as 32bit on the Pi Zero.

**Comment by pokitoz** [comment]
If that still interests you, you can have a look at:
https://github.com/ros2-for-arm/ros2/issues/7
